js/block.js

```diff
@@ -639,6 +718,22 @@ var convert = {
     any: function(text){return util.isNumber(text) ? +text : text; }
 };
 
+/*****************
+*
+*  wb-contains
+*
+*  Instantiated as new WBContains or as <wb-contains>. Contains is a semi-block. It never
+*  exists by itself, only as a child of a <wb-context> (and every <wb-context has at least one
+*  <wb-contains> child, implicitly if not explicitly). Sometimes we need to test for other blocks
+*  being in a <wb-context>, which is why sometimes it is block-ish.
+*
```

Comment:
`execution.js` currently relies on this fact. I say, the bloat of having `<wb-contains>` doesn't hurt that much right now.